### ISSUE TITLE
op_label_encoder support for keys_strings & values_floats 

### ISSUE THREAD

**Issue by ebolotin6** [sdpython/mlprodict#335]
Hello,

Could you please update `mlprodict/onnxrt/ops_cpu/ops_label_encoder.py` to include support for the `keys_strings`, `values_floats`, and `default_float` combination? I need this use case for a transformer that converts categorical columns into floats. 

Concrete update:
```
        elif len(self.keys_strings) > 0 and len(self.values_floats) > 0:
            self.classes_ = {k.decode('utf-8'): v for k, v in zip(
                self.keys_strings, self.values_floats)}
            self.default_ = self.default_float
            self.dtype_ = numpy.float32
```

If acceptable, then I can push my local branch and open an MR.

Thank you

**Comment by xadupre** [comment]
Feel free to do it. I'll merge it.

**Comment by sdpython** [comment]
Merged in #339. Feel free to reopen.
